.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+name: ci
+
+on:
+  push:
+    branches: [ master ]
+    tags:
+      - 'v*'
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  publish:
+    if: startsWith(github.ref, 'refs/tags/v')
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.17
+
+    - name: Publish
+      run: ./publish.sh
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
You have to set the `VERSION` environment variable here. Judging from the `publish.sh` it will fail otherwise.

Reply:
While you are at it maybe also remove the `TRAVIS_TAG` variable from `publish.sh`

Reply:
Forget the `VERSION` comment (I overlooked that it is generated) but how about reflecting that in the workflow and only trigger on tags? Otherwise the build will just cost us money and not do anything.

Reply:
Bonus points for documenting the steps to release a new version ;)

Reply:
can't find `TRAVIS_TAG` in the publish script 😕

Reply:
how about this?

https://github.com/Jimdo/vault-unseal/blob/master/publish.sh#L9-L12

Reply:
that's on master, but it's fixed exactly in this PR

Reply:
I do not see any change to the publish.sh script in this branch ...